### DISCUSSION TITLE
Web UI has no icons

### DISCUSSION THREAD

**Discussion by wilbert-alberts** [zwave-js/zwave-js-ui#2567] [Q&A] [answered]
Hi,

I'm trying to met zwavejs2mqtt up and running in a docker environment. I succeeded once but had to take it down. Since then, whenever I open the web interface, the icons are gone. Instead the 'alternative' text is shown. This removes the U from the UI. (In other words: the page becomes unreadable/unusable.) Any idea what might be wrong?

Regards,
Wilbert

**Comment by robertsLando** [comment, marked as answer]
Use 6.15.1

**Reply by wilbert-alberts** [reply]
Thanks! (Did I miss that in the discussions somewhere?)

**Reply by robertsLando** [reply]
There have been 2/3 issues about this and another discussion but no problem it has been my fault, sorry for that :)
